config/locales/en.yml

```diff
@@ -402,10 +402,10 @@ en:
           title: Get benefits and support
           href: /check-benefits-financial-support
           image_src: homepage/cost-of-living.jpg
-        - text: Find out about the UK's response to the Russian government's invasion of Ukraine and how you can help.
-          title: War in Ukraine
-          href: https://ukstandswithukraine.campaign.gov.uk/
-          image_src: homepage/ukraine-promo.jpg
```

Comment:
We [have a convention](https://github.com/alphagov/frontend/blob/main/docs/update-homepage-promotion-slots.md#4-delete-the-previous-images) to delete this file

Reply:
Thanks, addressed in 1cac5b1